docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -38,7 +38,7 @@ A software-defined asset includes the following:
 | Name                                  | Description                                                                                                                                                                                                                  |
 | ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | <PyObject object="asset" decorator /> | A decorator used to define assets.                                                                                                                                                                                           |
-| <PyObject object="SourceAsset" />     | A class that describes an asset, but doesn't define how to compute it. <PyObject object="SourceAsset" />s are used to represent assets that other assets depend on, in settings where they can't be materialized themselves. |
+| <PyObject object="SourceAsset" />     | A class that describes an asset, but doesn't define how to compute it. <PyObject object="SourceAsset" />s are used to represent assets that other assets or jobs depend on, in settings where they can't be materialized themselves. |
```

Comment:
I think this problem might pre-exist this change, but is this definition in the context of the global asset lineage graph?

Reply:
Let me know if this doesn't answer your question:

My thinking behind the definition here is that the "source asset" designation is setting-dependent. An asset isn't fundamentally a source asset (because all data is generated by code in some way). Rather, `SourceAsset` is a class that's used to represent an asset in settings where it's not necessary to know how the asset is computed:
- In a job that reads from the asset but doesn't materialize it
- In a `Definitions` object / code location
- There are no Python classes in play when looking at the global asset graph in Dagit, but I think it's reasonable to use the term "source asset" in that context for assets that don't have a materialization functions defined in any code location

Reply:
Great that all makes sense to me. 👍🏻